pipelines-scripts/release.ps1

```diff
@@ -25,7 +31,10 @@ Get-AzContext
 
 $importedBp = Get-AzBlueprint -ManagementGroupId $mgId -Name $blueprintName -LatestPublished
 # Urgent TODO - this should be idemopotent...
-# todo - should auto-insert blueprintId into parameters file
+
```

Comment:
This looks good, but I need to test it. Thank you for taking care of this!

Reply:
I've changed a variable name in the last commit.